packages/babel-plugin-transform-do-expressions/README.md

```diff
@@ -75,14 +75,14 @@ const Component = props =>
   <div className='myComponent'>
     {do {
       if(color === 'blue') { <BlueComponent/>; }
-      if(color === 'red') { <RedComponent/>; }
-      if(color === 'green') { <GreenComponent/>; }
+      else if(color === 'red') { <RedComponent/>; }
+      else if(color === 'green') { <GreenComponent/>; }
     }}
   </div>
 ;
 ```
 
-[Try in REPL](http://babeljs.io/repl/#?evaluate=true&presets=es2015%2Creact%2Cstage-0&code=const%20Component%20%3D%20props%20%3D%3E%0A%20%20%3Cdiv%20className%3D'myComponent'%3E%0A%20%20%20%20%7Bdo%20%7B%0A%20%20%20%20%20%20if(color%20%3D%3D%3D%20'blue')%20%7B%20%3CBlueComponent%2F%3E%3B%20%7D%0A%20%20%20%20%20%20if(color%20%3D%3D%3D%20'red')%20%7B%20%3CRedComponent%2F%3E%3B%20%7D%0A%20%20%20%20%20%20if(color%20%3D%3D%3D%20'green')%20%7B%20%3CGreenComponent%2F%3E%3B%20%7D%0A%20%20%20%20%7D%7D%0A%20%20%3C%2Fdiv%3E%0A%3B)
```

Comment:
@hzoo so I reverted the change on that line, not sure if it is what you meant (or deleting all [Try in REPL] lines in that file?)